homeassistant/brands/aeonmatrix.json

```diff
@@ -0,0 +1,5 @@
+{
+  "domain": "aeonmatrix",
+  "name": "Aeon Matrix",
+  "integrations": ["yardian"]
+}
```

Comment:
We usually don't make a brands file until there are multiple integration from the same vendor

Reply:
got it